airbyte-ci/connectors/live-tests/src/live_tests/commons/connector_runner.py

```diff
@@ -150,15 +151,15 @@ async def _run(
                     "sh",
                     "-c",
                     " ".join(airbyte_command)
-                    + f"| {self.IN_CONTAINER_OBFUSCATOR_PATH} > {self.IN_CONTAINER_OUTPUT_PATH} 2>&1 | tee -a {self.IN_CONTAINER_OUTPUT_PATH}",
+                    + f"| {self.IN_CONTAINER_OBFUSCATOR_PATH} 2>{self.IN_CONTAINER_STDERR_PATH} 2>&1 | tee -a {self.IN_CONTAINER_STDOUT_PATH}",
```

Comment:
We're redirectiong the obsfuscator and not the connector output right?
I think that in any case the use of `|` after the connector command means that the obsfuscator only gets `stdout`. 
I think we should do something like this instead:
```
command > >(obfuscator consumes stdout) 2> >(obfuscator consumes stderr)
```